allure-web/src/components/execution/steps-list.hbs

```diff
@@ -1,26 +1,31 @@
 {{#each steps}}
     <div class="step">
-        <div class="{{b "step" "title" hasContent=(or steps.length attachments.length)}}">
-            {{#if (or steps.length attachments.length)}}
+        <div class="{{b "step" "title" hasContent=(or steps.length attachments.length parameters.length)}}">
+            {{#if (or steps.length attachments.length parameters.length)}}
                 <span class="step__arrow block__arrow">{{arrow status}}</span>
             {{else}}
                 <span class="step__status">{{allure-icon status}}</span>
             {{/if}}
+            <div class="step__name">
             {{name}}
-            {{#if parameters}}
-                [
-                {{#each parameters}}
-                    <span>{{name}}: {{value}}</span>
-                {{/each}}
-                ]
-            {{/if}}
+            </div>
```

Comment:
Currently it doesn't encourage write good steps, but does a lot to fit weird use-cases.